lineage/figures/figure6.py

```diff
@@ -40,7 +40,15 @@ def accuracy():
     """ A Helper function to create more random copies of a population. """
     # Creating a list of populations to analyze over
     list_of_Es = [[StateDistribution(E2[1].params[0], E2[1].params[1], E2[1].params[2], a, E2[1].params[4], E2[1].params[5]), E2[1]] for a in np.linspace(4.0, 20.0, num_data_points)]
-    list_of_populations = [[LineageTree.init_from_parameters(pi, T, E, max_desired_num_cells)] for E in list_of_Es]
+    list_of_populations = []
```

Comment:
The abundance of the cell states should be changed by adjusting pi and T, not by filtering out lineages.